### ISSUE TITLE
Duplicate Optae in AL HQ data

### ISSUE THREAD

**Issue by Jonathan-Blazeby** [BSData/horus-heresy#3234]
**Describe the bug**
18 Identical unit entries for Optae appearing in the HQ category for the Alpha Legion data
![image](https://github.com/BSData/horus-heresy/assets/71888333/3a2a6887-d82a-4767-b26c-a9b04c7041eb)

**To Reproduce**
1. Go to LA -XX: Alpha Legion new Crusade FOC
2. See multiple Optae in HQ section on left

**Expected behavior**
Only one Optae unit entry in HQs

**Screenshots**

**Device and data (please complete the following information):**
 - OS: PC Windows 10
 - App being used to create lists: Battlescribe
 - App version: 2.03.21
 - Data release version v19.0.3

**Additional context**

**Comment by The4D6** [comment]
Yeah, the import box is checked in every cat, so the AL cat imports all of them
